app/src/main/java/com/keylesspalace/tusky/worker/PruneCacheWorker.kt

```diff
@@ -41,6 +48,8 @@ class PruneCacheWorker(
         return Result.success()
     }
 
+    override suspend fun getForegroundInfo() = ForegroundInfo(NOTIFICATION_ID_PRUNE_CACHE, notification)
```

Comment:
Why here as well? This one is not expedited?

Reply:
Belt and braces. If future code launches this as an expedited worker it won't work without this.

But the failure will be effectively silent (there's no user notification that it happened), and unless you're (a) checking the logs when it happens, or (b) notice that the database is growing without bounds, it won't be noticed.

Reply:
ok